### PR TITLE
Git feature loading screen

### PR DESCRIPTION
**Pull request by folkertdev**
representing what the state of the attribution is with RemoteData

### CODE REVIEW

**Review by LuigiGao — APPROVED** [review]

File: frontend/src/Update.elm [hunk 1]
```diff
@@ -17,6 +17,7 @@ import Dict exposing (Dict)
 import Types exposing (..)
 import InputField exposing (OutMsg(..))
 import Ports
+import Process
```
Comment (folkertdev) [inline frontend/src/Update.elm]: can be removed

File: frontend/src/Types.elm [hunk 2]
```diff
@@ -50,10 +50,15 @@ type AttributionMessage
     = SetLanguage Language
     | SetFeatureCombo FeatureCombo
     | PerformAttribution
+    | CancelAttribution
     | ServerResponse (Result Http.Error AttributionResponse)
     | AttributionInputField Author InputField.Msg
 
 
+type alias Id =
```
Comment (folkertdev) [inline frontend/src/Types.elm]: can be removed

File: frontend/src/Update.elm [hunk 3]
```diff
@@ -275,6 +283,14 @@ performAttribution attribution =
             |> Http.send ServerResponse
 
 
+
+{-
```
Comment (folkertdev) [inline frontend/src/Update.elm]: can be removed

File: frontend/src/View.elm [hunk 4]
```diff
@@ -199,6 +220,7 @@ attributionView attribution =
                     , unknownAuthorInput
                     ]
                 ]
+            , isLoading {- div [ class "overlay" ] [ div [ class "loading-prompt" ] [] ] -}
```
Comment (folkertdev) [inline frontend/src/View.elm]: is that comment needed?

**Review by folkertdev — COMMENTED** [review]
some style things that should be addressed.

**Pull request merged**